src/storage/src/compaction/picker.rs

```diff
@@ -69,7 +94,18 @@ impl<S: LogStore> Picker for SimplePicker<S> {
         req: &CompactionRequestImpl<S>,
     ) -> crate::error::Result<Option<CompactionTaskImpl<S>>> {
         let levels = &req.levels();
+        let expired_ssts = self
+            .get_expired_ssts(levels, req.ttl)
+            .map_err(|e| {
+                error!(e;"Failed to get region expired SST files, region: {}, ttl: {:?}", req.region_id, req.ttl);
+                e
+            })
+            .unwrap_or(vec![]);
 
+        info!(
```

Comment:
If the `expired_ssts` is empty, the log may be annoying. Only log it when it's not empty would be better.